internal/move2kubeapi/move2kubeapi.go

```diff
@@ -159,13 +163,18 @@ func getAsset(w http.ResponseWriter, r *http.Request, isCustomization bool) {
 	w.Header().Set("Content-Type", "application/octet-stream")
 	w.Header().Set("Content-Disposition", "attachment; filename="+filename)
 	if _, err := io.Copy(w, file); err != nil {
-		log.Errorf("Error while getting asset : %s", err)
+		log.Errorf("failed to send the asset/customization to the client. Error: %q", err)
 	}
 }
 
 func deleteAsset(w http.ResponseWriter, r *http.Request, isCustomization bool) {
 	name := mux.Vars(r)["name"]
-	asset := mux.Vars(r)["asset"]
+	asset := ""
+	if isCustomization {
+		asset = mux.Vars(r)["customization"]
```

Comment:
Since we are using the string in multiple places, do we want to change it to a const?

Reply:
hmm, there are lot of strings like `asset`, `name`, etc. that are also hardcoded. Should we change everything to be const?

Reply:
if there is a name referenced in more than one place, and it needs to be consistent, then yes.